src/coord/src/catalog/builtin.rs

```diff
@@ -243,6 +243,16 @@ lazy_static! {
         id: GlobalId::System(2011),
         index_id: GlobalId::System(2012),
     };
+    pub static ref MZ_SCHEMAS: BuiltinTable = BuiltinTable {
+        name: "mz_schemas",
+        desc: RelationDesc::empty()
+            .with_column("global_id", ScalarType::String.nullable(false))
```

Comment:
This is actually not a `global_id`; it's an ID in a totally separate namespace. Maybe `schema_id`?

Reply:
oops -yes!